lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/base.good

```diff
@@ -0,0 +1,27 @@
+created
+uncreate
+uncreated
+imply
+implied
+unnatural
+conveyed
+sawyer
+NASA
+FAQs
+can't
+doesn't
+won't
+Created
+Hello
+HELLO
+NASA
+etc.
+etc
+HELLO
+lip.
+text.
+NASA.
+Text.
+TEXT.
```

Comment:
What happens with a word ending in ellipsis (...)? Should we add a test check for this here too? I also wonder if hunspell handles the ellipsis glyph (unicode character) somehow or it's treated as a character (just curiosity)?

Reply:
I'll add a test with `...`. AFAIU Hunspell's code has no special treatment for ellipsis glyph, but it shouldn't be an issue as it should be ignored by tokenizer. And dots in some languages are considered to be word parts.

Reply:
Yeah, I was just wondering how English-centric this is.

> And dots in some languages are considered to be word parts.

Can you give me an example? Interesting!

Reply:
French, see https://github.com/wooorm/dictionaries/blob/main/dictionaries/fr/index.aff#L13. I'm not sure why that is, but it is :)

For now, I'm leaving `WORDCHARS` treatment out of Lucene's Hunspell as it only affects tokenizer, and my understanding is that Lucene tokenizes itself somehow. I wonder if this is indeed right.

Reply:
Let's ask folks who speak French - @jpountz would you know of a word with dots inside? Bag.uette? :)

Lucene tokenizers are indeed completely independent. A normalization filter can handle odd glyphs and normalize them into something more palatable.